web/source/kmwembedded.js

```diff
@@ -480,8 +481,9 @@
    *  
    *  @param  {number}  code   key identifier
    *  @param  {number}  shift  shift state (0x10=shift 0x20=ctrl 0x40=alt)
+   *  @param  {number}  lstates lock state (caps, scroll, num locks)
```

Comment:
Document values for lstates

Reply:
Oh, the name is inconsistent with `lstate` above, perhaps both should be `lstates`